src/test/java/org/broadinstitute/hellbender/tools/walkers/filters/VariantFiltrationIntegrationTest.java

```diff
@@ -239,4 +239,14 @@ public void testSetVcfFilteredGtoNocall()  throws IOException {
 
         spec.executeTest("testSetVcfFilteredGtoNocall", this);
     }
+
+    @Test
+    public void testFilteringDPfromFORMATAndFailMissing() throws IOException {
+        final IntegrationTestSpec spec = new IntegrationTestSpec(
+                baseTestString("filteringDepthInFormatWithMissing.vcf", "  --missingValuesInExpressionsShouldEvaluateAsFailing --genotypeFilterExpression 'DP < 10 || DP == 22' --genotypeFilterName lowDP "),
```

Comment:
@ronlevine I think this test should tests something other than DP.  Since DP is in the INFO field as well, there is strange fallback behavior if it's not found as a genotype annotation.  Could you change the expression to test something else, like Z, that isn't in the info field?